domain/application/service/unit.go

```diff
@@ -131,10 +131,16 @@ type UnitState interface {
 	// IsSubordinateApplication returns true if the application is a subordinate application.
 	IsSubordinateApplication(context.Context, coreapplication.ID) (bool, error)
 
-	// GetMachineNetNodeUUIDFromName returns the net node UUID for the named machine.
-	// The following errors may be returned:
-	// - [applicationerrors.MachineNotFound] if the machine does not exist
+	// GetMachineNetNodeUUIDFromName returns the net node UUID for the named
+	// machine. The following errors may be returned: -
+	// [applicationerrors.MachineNotFound] if the machine does not exist
```

Comment:
Comment is broken

Reply:
Keep to 80 line width, or I'll auto format it.